### PR TITLE
Docs for basename

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -2618,6 +2619,13 @@ Given an array of optional values, `select_all` will select only those elements
 
 This function will return `false` if the argument is an unset optional value. It will return `true` in all other cases.
 
+## String basename(File)
```
Comment (Horneth) [inline SPEC.md]: does it only work on a `File` or will it work on a `String` too ? (Or anything that coerces to a `File`)

**Comment by Horneth** [comment]
👍

File: SPEC.md (hunk 1 above)
Comment (cjllanwarne) [inline SPEC.md]: Technically I guess it's anything that can coerce into `String`

**Comment by mcovarr** [comment]
:+1:

**Pull request merged**